### ISSUE TITLE
Action Control Panel Robot Test Fails Randomly

### ISSUE THREAD

**Issue by jensens** [plone/Products.CMFPlone#1466]
with Page should have contained element 'xpath=//section[@role='menu']/section/ul/li[1]/a/span[contains(text(), 'Accessibility')]' but did not

I suppose this is because 
https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/tests/robot/test_controlpanel_actions.robot#L92
does not wait for page load.

cc @ebrehault

**Comment by thet** [comment]
@jensens the failure occurs for 5.1, or not?
it's most likely because of the merge of https://github.com/plone/Products.CMFPlone/pull/1431, where the footer portlet ``section`` tag was changed to an ``aside``.

**Comment by jensens** [comment]
@thet ok. so why wasnt that catched by PR testing? Anyway its a good idea to Wait :)

**Comment by thet** [comment]
I don't why this failure happened... where did it happen?

**Comment by ebrehault** [comment]
I on it.
@jensens: wait for something to appear is only useful if the rendering is produced by javascript. If the expected text is supposed to be part of the initially loaded DOM, RobotFramework will find it.

**Comment by jensens** [comment]
> I don't why this failure happened... where did it happen?

@agitator had it, but he re-run the test already, so its not visible in jenkins anymore

**Comment by jensens** [comment]
@ebrehault thanks for explanation, I was not aware of this detail.

**Comment by ebrehault** [comment]
The xpath was wrong indeed, I made a PR to fix it.

**Comment by gforcada** [comment]
@ebrehault I already fixed that during the weekend https://github.com/plone/Products.CMFPlone/commit/69438696d4c5b03c2e37889de05a3cf3cfd36e8f is it broken again on 5.1?

**Comment by ebrehault** [comment]
@gforcada, oh, well I pulled the current master and it was not there. Is it merged?

**Comment by ebrehault** [comment]
@gforcada I am wrong: your commit was there, but the path you have used is wrong (there is no `aside` anymore)

**Comment by ebrehault** [comment]
@gforcada sorry my mistake, I was on 5.0

**Comment by ebrehault** [comment]
Ok my version works on bothe 5.0 and 5.1, I merge it.

**Comment by ebrehault** [comment]
@gforcada since my fix works on both 5.0 and 5.1, I merge it
